clients/fides-js/src/fides-tcf.ts

```diff
@@ -101,7 +101,8 @@ declare global {
       parameter?: number | string
     ) => void;
     config: {
-      fides: OverrideOptions;
+      // TODO: make configurable!
```

Comment:
NOTE: I think the *right* solution here is to default to `window.fides_config` as an override object, but also support the ability to pass the override object into `Fides.init`, e.g.
```
window.config.my_object = { fides_string: "123" };
// later...
window.Fides.init(overrideOptions: window.config.my_object);
```